bots/task/github.py

```diff
@@ -55,6 +55,7 @@
 # it will publish a test task to the queue (used to trigger specific contexts)
 NOT_TESTED_DIRECT = "Not yet tested (direct trigger)"
 
+# TODO this is not gonna work, is it?
```

Comment:
Now we won't be able to use this `OUR_CONTEXTS`, since tests are going to start always with `TEST_OS`. @martinpitt mentioned something about this and that he talked about it with @Gundersanne. What was the agreed solution?

Reply:
I had something else in mind when I said that we shouldn't need a static "cockpit/" prefix. Indeed we'll have to rework this. `known_context()` is used in tests-trigger as a sanity check to prevent us trying to e. g. retry semaphore with this (which won't work). But the more important user is `statuses()`.

So either `known_context()` checks if the first part (`.split('/')[0]`) exists (as link) in bots/images/ (some caching should be involved), or we have to use a static cockpit/ prefix. I'm just a bit concerned about test names getting too long, with the extra /branch name that we want to tack at the end. We could shorten it to "c/"..

But WDYT about the "is this a known image" check? It's not that expensive and easy to do.

Reply:
> But WDYT about the "is this a known image" check? It's not that expensive and easy to do.

I agree that checking if the first part is valid image makes sense. Adding `cockpit` could make some names way too long and it is still just noise. The same goes with `c/`. I'll take a look at this and send some commit.